### PR TITLE
Move TabbedBox from unstable to core components as Tabs

### PR DESCRIPTION
**Pull request by xginn8**
HQ: balena-io/balena#1663
Change-type: minor
Signed-off-by: Matthew McGinn <mamcgi@gmail.com>

### CODE REVIEW

**Comment by xginn8** [comment]
Demo screenshots attached 
rendition-tab-2
rendition-tab-1

File: src/components/TabbedBox.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	children: JSX.Element[];
+}
+
+interface TabbedBoxState {
+	show: number;
+}
+
+const FlexTab = styled(Flex)`
+	border-style: hidden;
+	border-bottom: black;
```
Comment (LucianBuzzo) [inline src/components/TabbedBox.tsx]: Can you please change the border color to match the border color we use on textareas and inputs?

Comment (xginn8) [reply src/components/TabbedBox.tsx]: Done

**Comment by sradevski** [comment]
@xginn8 @LucianBuzzo Any good reason why we should move it to core components? I think what we talked about basing rendition on Grommet will bring us better Tabs out of the box, so I would leave this in the unstable components until then.

**Comment by LucianBuzzo** [comment]
@sradevski This is needed right now to add some much needed functionality to the balenaCloud dashboard. Moving it into core seems fine for now considering the narrow scope of the component functionality.

**Comment by sradevski** [comment]
@LucianBuzzo What I meant is maybe we can just use it from the unstable set of components, but it's also fine if we move it to the core components I guess. We will have a breaking change when moving to grommet anyway, so we can fix it then.

**Comment by LucianBuzzo** [comment]
@sradevski `Tabs` is a much better name - @xginn8 want to update this component to use the simpler name?

**Comment by xginn8** [comment]
> @sradevski `Tabs` is a much better name - @xginn8 want to update this component to use the simpler name?

Updated the name everywhere to `Tabs`, thanks!

**Comment by xginn8** [comment]
@resin-ci retest

**Pull request merged**